<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDataLogFileWidgetUI</class>
 <widget class="QWidget" name="ProfileDataLogFileWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>80</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="qlineedit_scheme">
     <property name="placeholderText">
      <string>Please set a scheme</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="kpushbutton_scheme"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_scheme">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Name/Scheme:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_timestamps">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Write Media Catalog Number (MCN) and International Standard Recording Codes (ISRC, ISO 3901) of the tracks into the cue sheet (if your drive have the capability).&lt;/p&gt;&lt;p&gt;Depending in your drive it could take a few seconds at the end of your ripping process to fetch those numbers and codes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>Write timestamps</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
